### PR TITLE
EVE-817 - Setting up EVE pipeline

### PR DESCRIPTION
**Pull request by tcarmet**

### CODE REVIEW

**Comment by ThibaultRiviere** [comment]
I think you can close the others PR, once merged, you will have to forward this one until master

**Comment by tcarmet** [comment]
@ThibaultRiviere Ok! will do!

**Comment by tcarmet** [comment]
@ThibaultRiviere all the other PR are now closed. We can merge this PR if everything sounds good to you and your team.

File: eve/main.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+          command: npm run --silent lint_md
+      - ShellCommand:
+          name: run test
+          command: npm run --silent test
+      - ShellCommand:
+          name: run ft_test
+          command: npm run ft_test
+      - ShellCommand:
+          name: run executables tests
+          command: npm install && npm test
```
Comment (ThibaultRiviere) [inline eve/main.yml]: Why an other npm install ?
Should we change the order ? like first `npm run --silent test` and then `npm run ft_test`

Comment (tcarmet) [reply eve/main.yml]: Actually the `npm install && npm test`  was defined in the `master` version of the `circle.yml`

Indeed it's not available in the 6.4 version so I don't have any good explanation. But it works so I've decided to leave it. (we can remove it in this version if you think it doesn't make sense here).

If you take a look at the master version, there's the following line:
```shell
cd lib/executables/pensieveCreds && npm install && npm test
```
Which correspond to this step.

**Comment by DavidPineauScality** [comment]
@ironman-machine r+

File: eve/main.yml (hunk 1 above)
Comment (DavidPineauScality) [reply eve/main.yml]: MHh this actually appeared much later than 6.4, and should be added afterwards on later versions/branches.

**Comment by DavidPineauScality** [comment]
@ironman-machine r-

File: eve/main.yml (hunk 1 above)
Comment (ThibaultRiviere) [reply eve/main.yml]: As discuss offline. Let's remove it here, forward this change and create and other PR targeting rel/7.2 to add it where it make sens . 
Thanks :)

**Comment by tcarmet** [comment]
@ThibaultRiviere @DavidPineauScality I've removed the requested part.

**Comment by DavidPineauScality** [comment]
@ironman-machine r+

**Comment by tcarmet** [comment]
need your approval @DavidPineauScality

**Comment by DavidPineauScality** [comment]
Merging manually to avoid a full re-cycle; my bad ~!

**Comment by tcarmet** [comment]
Thank you :heart:

**Pull request merged**